e2e-tests/fixtures/Plan.ts

```diff
@@ -555,7 +574,7 @@ export class Plan {
 export enum PanelNames {
   ACTIVITY_DIRECTIVES_TABLE = 'Activity Directives Table',
   SIMULATED_ACTIVITIES_TABLE = 'Simulated Activities Table',
-  ACTIVITY_TYPES = 'Activity Types',
+  ACTIVITY_AND_RESOURCE_TYPES = 'Activity & Resource Types',
```

Comment:
Should the name of the field be `TIMELINE_ITEMS`, instead of `ACTIVITY_AND_RESOURCE_TYPES`, to more closely reflect the view values in `ViewGridComponent` just so that it's less of a mental jump to try to match them?